### PR TITLE
Kraken: add metrics for data loading

### PR DESCRIPTION
**Pull request by kinnou02**

### CODE REVIEW

**Review by TeXitoi — APPROVED** [review]

File: source/kraken/maintenance_worker.h [hunk 1]
```diff
@@ -40,12 +40,16 @@ www.navitia.io
 
 namespace navitia {
 
+class Metrics;
```
Comment (benoit-bst) [inline source/kraken/maintenance_worker.h:43]: Or you can just add the `#include "metrics.h` here, instead of using _forward declaration._  
As you wish.

**Review by benoit-bst — APPROVED** [review]

File: source/kraken/maintenance_worker.h (hunk 1 above)
Comment (TeXitoi) [reply source/kraken/maintenance_worker.h:43]: forward declare in .h limit compilation times.

Comment (benoit-bst) [reply source/kraken/maintenance_worker.h:43]: Well, it it true that navitia has a long long compilation time... My mistake

File: source/kraken/metrics.h [hunk 2]
```diff
@@ -69,10 +69,17 @@ class Metrics: boost::noncopyable {
...
+    prometheus::Histogram* data_loading_histogram;
+    prometheus::Histogram* data_clonning_histogram;
+    prometheus::Histogram* handle_rt_histogram;
 public:
     Metrics(const boost::optional<std::string>& endpoint, const std::string& coverage);
     void observe_api(pbnavitia::API api, double duration) const;
     InFlightGuard start_in_flight() const;
+
+    void observe_data_loading(double duration) const;
+    void observe_data_clonning(double duration) const;
```
Comment (pbougue) [inline source/kraken/metrics.h]: :gb: typo: cloning (one `n`)

File: source/kraken/metrics.h [hunk 3]
```diff
@@ -69,10 +69,17 @@ class Metrics: boost::noncopyable {
     std::shared_ptr<prometheus::Registry> registry;
     std::map<pbnavitia::API, prometheus::Histogram*> request_histogram;
     prometheus::Gauge* in_flight;
+    prometheus::Histogram* data_loading_histogram;
+    prometheus::Histogram* data_clonning_histogram;
```
Comment (pbougue) [inline source/kraken/metrics.h]: :gb: typo: cloning (please ctrl+F over this PR)

**Pull request merged**